spacy/training/loop.py

```diff
@@ -87,9 +87,12 @@ def train(
             if is_best_checkpoint is not None and output_path is not None:
                 with nlp.select_pipes(disable=frozen_components):
                     update_meta(T, nlp, info)
-                with nlp.use_params(optimizer.averages):
-                    nlp = before_to_disk(nlp)
```

Comment:
The `before_to_disk` callback should be preserved in these changes.

Reply:
Done. I did not called before_to_disk in the save below because this one will always be called